word-search.py

```diff
@@ -212,9 +216,106 @@ def randomizer(grid):
 
     return grid
 
-def generateWordList():
-    # compiles the list of words that the user can find in the game
-    return
+# creates a dictionary of all distinct letters in the grid as keys
+# and the list of all their coordinate locations as the value
+def find_starting_coordinates(grid):
+    startingCoordinates = {}
+    for x in range(0, len(grid)):
+        for y in range(0, len(grid[x])):
+            if grid[x][y] in startingCoordinates.keys():
+                startingCoordinates[grid[x][y]].append([x, y])
+            else:
+                startingCoordinates[grid[x][y]] = [[x, y]]
+    return startingCoordinates
+
+def find_next_tile(nextLetter, xyCoordinates, grid, usedTiles):
+    x = xyCoordinates[0]
+    y = xyCoordinates[1]
+    maxIndex = len(grid)-1
+    matches = []
+
+    # checks each of the surround tiles if it exists,
+    # if it has not yet been used to form the word,
+    # and if is equal to the next letter in the key/word
+    if (x-1 >= 0 and y-1 >= 0 and (not [x-1, y-1] in usedTiles) and nextLetter == grid[x-1][y-1]):
```

Comment:
Just a suggestion 😁 maybe we can put yung mga x-1 etc sa variable para less intimidating lang 😅 
e.g.
    up = x-1
    down = x+1
    left = y-1
    right = y+1

Reply:
Haha, sure. If it makes the code easier to read, will update.